app/services/search_context.rb

```diff
@@ -27,7 +27,7 @@ def initialize(current_user, query = {})
   # *** Method that outputs the next step for the user to complete its rdv journey ***
   # *** It is used in #to_partial_path to render the matching partial view ***
   def current_step
-    if address.blank? && @organisation_id.blank?
+    if address.blank? && @organisation_id.blank? && @service_id.blank?
```

Comment:
C'est peut-être plutôt le territoire qui est la clef (tant que nous n'avons pas « scopé » les services sur les territoires).
```suggestion
    if address.blank? && @organisation_id.blank? && @territory_id.blank?
```

Ou un truc dans le genre. L'adresse ou l'organisation servant finalement à définir un territoire (voir une zone géographique plus petite que le territoire via l'organisation ou la sectorisation)

Reply:
Je me suis fait la même réflexion en lisant cette ligne !